packages/terra-alert/src/_variables.scss

```diff
@@ -0,0 +1,71 @@
+$terra-alert-alert-color-scheme: (
```

Comment:
The CSS module migration looks good. My only comment would be I'm not sure there is much value in having these SASS mappings being its only 1prop. Perhaps we just assing the CSS property and variable directly to the class name. Thoguhts?